model/k8s/metrics.yaml

```diff
@@ -469,3 +469,77 @@ groups:
     attributes:
       - ref: k8s.namespace.phase
         requirement_level: required
+
+  # k8s.container.* metrics
+  - id: metric.k8s.container.cpu.limit
+    type: metric
+    metric_name: k8s.container.cpu.limit
+    stability: development
+    brief: "Maximum CPU resource limit set for the container"
+    note: |
+      See https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.30/#resourcerequirements-v1-core for details.
+    instrument: gauge
+    unit: "{cpu}"
+  - id: metric.k8s.container.cpu.request
+    type: metric
+    metric_name: k8s.container.cpu.request
+    stability: development
+    brief: "CPU resource requested for the container"
+    note: |
+      See https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.30/#resourcerequirements-v1-core for details.
+    instrument: gauge
+    unit: "{cpu}"
+  - id: metric.k8s.container.memory.limit
+    type: metric
+    metric_name: k8s.container.memory.limit
+    stability: development
+    brief: "Maximum memory resource limit set for the container"
+    note: |
+      See https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.30/#resourcerequirements-v1-core for details.
+    instrument: gauge
+    unit: "By"
+  - id: metric.k8s.container.memory.request
+    type: metric
+    metric_name: k8s.container.memory.request
+    stability: development
+    brief: "Memory resource requested for the container"
+    note: |
+      See https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.30/#resourcerequirements-v1-core for details.
+    instrument: gauge
+    unit: "By"
+  - id: metric.k8s.container.storage.limit
+    type: metric
+    metric_name: k8s.container.storage.limit
+    stability: development
+    brief: "Maximum storage resource limit set for the container"
+    note: |
+      See https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.30/#resourcerequirements-v1-core for details.
+    instrument: gauge
+    unit: "By"
+  - id: metric.k8s.container.storage.request
+    type: metric
+    metric_name: k8s.container.storage.request
+    stability: development
+    brief: "Storage resource requested for the container"
+    note: |
+      See https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.30/#resourcerequirements-v1-core for details.
+    instrument: gauge
+    unit: "By"
+  - id: metric.k8s.container.ephemeralstorage.limit
+    type: metric
+    metric_name: k8s.container.ephemeralstorage.limit
+    stability: development
+    brief: "Maximum ephemeral storage resource limit set for the container"
+    note: |
+      See https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.30/#resourcerequirements-v1-core for details.
+    instrument: gauge
+    unit: "By"
+  - id: metric.k8s.container.ephemeralstorage.request
+    type: metric
+    metric_name: k8s.container.ephemeralstorage.request
```

Comment:
why `ephemeralstorage` instead of `ephemeral_storage`? thanks

Reply:
I don't have a strong reason for doing so other than bringing this in from the [Collector implementation](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.111.0/receiver/k8sclusterreceiver/documentation.md#k8scontainerephemeralstorage_limit). (That's not a K8s API resource name)

On K8s the resource's name is `ephemeral-storage` (https://pkg.go.dev/k8s.io/api/core/v1#ResourceName) so it's up to us to decide.

Reply:
thanks, let's stick with semconv naming guidance then